docs/user-guide/gaffer-basics/what-is-a-graph.md

```diff
@@ -0,0 +1,47 @@
+# What is Graph?
+
+## Introduction to Graph Technology 
+
+Graph databases are built to allow you to store and navigate relationships. Their value lies in allowing you to analyse and gain insight into relationships between different data sources to see the bigger picture more easily than through tables or documents. They allow you to store and examine data in a flexible way, similar to how you might sketch ideas on a whiteboard or in a mind map. 
+
+Graph databases use nodes (also known as vertexes) to store data entities, and edges to store the relationships between these entities. An edge will always have a starting node, an end node, a type and a direction. They may describe parent-child relationships, actions, ownership and much more. There is no limit to the number and different types of relationships a node may have.
+
+Below is a simple example with two nodes representing people who have a relationship of 'friends' connecting them:
+
+```mermaid
+flowchart TD
+    subgraph Relationship
+        E((Person - Dave))
+        -- FRIENDS -->
+        F((Person - Chris))  
+    end
+```
+Figure 1 - Relationship between Dave and Chris.
```

Comment:
You can give a code block a title which might be preferable to this e.g. https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-a-title
```

Reply:
Wrapped the diagrams in example blocks instead. Title doesn't appear to work with mermaid.